src/ui/ScaleManager.js

```diff
@@ -36,8 +64,14 @@ var ScaleManager = exports = Class(function() {
 
 });
 
-ScaleManager.SCALE_MODE = {
-  NONE: "scaleModeNone",
-  LOCK_WIDTH: "scaleModeLockWidth",
-  LOCK_HEIGHT: "scaleModeLockHeight"
-};
\ No newline at end of file
+/**
+ * @var {object} ScaleManager.SCALE_MODE
+ * @property {string} NONE
+ * @property {string} LOCK_WIDTH
+ * @property {string} LOCK_HEIGHT
+ */
+exports.SCALE_MODE = {
```

Comment:
Like if I could just pass in `"LOCK_WIDTH"` or could alternatively use `scene.ScaleManager.LOCK_WIDTH`